deploy-service/common/src/main/java/com/pinterest/deployservice/bean/BuildBean.java

```diff
@@ -194,7 +195,7 @@ public SetClause genSetClause() {
         clause.addColumn("build_id", build_id);
         clause.addColumn("build_name", build_name);
         clause.addColumn("artifact_url", artifact_url);
-        clause.addColumn("scm", scm);
+        clause.addColumn("scm", StringUtils.capitalize(scm));
```

Comment:
modified this to unsure the capitalized value is what gets committed to the database